examples/app/pivot/theme.yaml

```diff
@@ -0,0 +1 @@
+
```

Comment:
If there aren't any customizations made in this file, it can just be omitted entirely.